server/utils/__tests__/date.spec.js

```diff
@@ -116,3 +117,33 @@ describe('getDateSelectionFrom', () => {
     ]);
   });
 });
+
+describe('getOffsetUnixTime', () => {
+  let NOW;
+  let NOW_MILLISECONDS;
```

Comment:
Sorry my fault, this could just be a const.
```
const NOW_MILLISECONDS = 1577836800;
```

Reply:
Agreed, also made NOW a constant as this is the same value in each test.

Reply:
So the date could be mutated so I'd leave that as a _let_ because we're risking test bleed. I don't think you can _freeze_ a date object? Well not in a conventional way.